nix/nixpkgs.nix

```diff
@@ -43,48 +43,6 @@ let
           wrapProgram $out/bin/pg_tmp --prefix PATH : ${pkgs.postgresql}/bin:$out/bin
         '';
       });
-      buildBazelPackage = pkgs.callPackage ./overrides/buildBazelPackage {
```

Comment:
Why is it safe to remove this override on MacOS? Is it the default in 19.03 now?

Reply:
Yep, afaict all our patches have since been fixed upstream.